packages/mrtd/src/DidCommMrtdModule.ts

```diff
@@ -53,4 +53,27 @@ export class DidCommMrtdModule implements Module {
       }),
     )
   }
+
+  /**
+   * Initializes the module (e.g. by pre-loading the CSCA Master List if configured).
+   */
+  public async initialize(agentContext: AgentContext): Promise<void> {
+    const logger = agentContext.config.logger
+
+    if (!this.config.masterListCscaLocation) {
+      logger.info('[DidCommMrtdModule] MASTER_LIST_CSCA_LOCATION not set; eMRTD verification remains disabled.')
+      return
+    }
+
+    try {
+      const csaMasterList = agentContext.dependencyManager.resolve(CscaMasterListService)
+      logger.debug('[DidCommMrtdModule] Preload CSCA MasterList...')
+      csaMasterList.initialize()
+      logger.debug(
+        `[DidCommMrtdModule] CSCA MasterList loaded (${csaMasterList.getTrustAnchors().length} CSCA anchors).`,
+      )
+    } catch (e) {
+      logger.error(`[DidCommMrtdModule] CSCA MasterList Preload failed: ${e instanceof Error ? e.message : String(e)}`)
+    }
```

Comment:
I think the try-catch won't work if not using the await. Something like that could work?

```suggestion
    try {
      const csaMasterList = agentContext.dependencyManager.resolve(CscaMasterListService)
      logger.debug('[DidCommMrtdModule] Preload CSCA MasterList...')
      csaMasterList.initialize()
  .then(() => {
    logger.debug(
      `[DidCommMrtdModule] CSCA MasterList loaded (${csaMasterList.getTrustAnchors().length} CSCA anchors).`
    )
  })
  .catch((e) => {
    logger.error(
      `[DidCommMrtdModule] CSCA MasterList Preload failed: ${e instanceof Error ? e.stack ?? e.message : String(e)}`
    )
  })
```